assets/js/main.js

```diff
@@ -0,0 +1,6 @@
+document.getElementById("addbtn").onclick = function(){
+    location.href= "seller.html";
```

Comment:
what about using `<a href="seller.html">` inside `index.html` file, instead of using javascript to change the location. since there is no functionality on the main page.